src/app/home/home.component.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
use the font-awesome npm package (https://www.npmjs.com/package/font-awesome) and I think you have to add the styles to angular.json under "styles" as well (use a relative path)